src/components/sitemap/index.ts

```diff
@@ -135,9 +239,8 @@ export class PostsSitemapIndex extends React.Component {
 export class ProfilesSitemapIndex extends React.Component {
```

Comment:
I think now it is useless code, can we remove?

Reply:
removed